lib/index.js

```diff
@@ -37,7 +38,8 @@ class ElectronDownloader {
   }
 
   get cache () {
-    return this.opts.cache || path.join(homePath(), './.electron')
+    // use passed argument or XDG environment variable fallback to OS default
+    return this.opts.cache || envPaths('electron-download', {suffix: ''}).cache
```

Comment:
Not sure since this is for the module `electron-download` thought it would make sense to name it like that... But since it's for `electron` it might make sense to name it like that.